pkg/kudoctl/packages/resolver/resolver_incluster_test.go

```diff
@@ -0,0 +1,171 @@
+package resolver
+
+import (
+	"log"
+	"testing"
+
+	"gotest.tools/assert"
```

Comment:
please switch to testify
```suggestion
	"github.com/stretchr/testify/assert"
```

Reply:
good catch